src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftVersionInfo.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```

Comment:
I'd strike this, as Sun has nothing to do with this file.

Reply:
there are refs to classes constructed at that time  ... I remember having this discussion with Clayton wrt when this plugin first came into being

Not going to diverge at this point